.github/workflows/build.sh

```diff
@@ -0,0 +1,32 @@
+#! /bin/bash
+
+# Usage:
+# ./.github/workflows/build.sh 
+
+# this script will modified upper directory istio-proxy repo
+# if self-used should attention that whether the upper directory have its own developing istio.proxy
+#first build should give permission to docker volumes
+#sudo chmod -R 777 /var/lib/docker/volumes
+
+UPDATE_BRANCH=${UPDATE_BRANCH:-"master"}
+cd ..
+rm -rf istio-proxy
+git clone -b ${UPDATE_BRANCH} https://github.com/intel/istio-proxy.git
+cp -rf istio/ istio-proxy/ 
```

Comment:
Is it necessary to copy istio as a subdirectory of istio-proxy? Can't they just be in separate directories?

Reply:
This is an effort made before to maintain only one build script. Now there is really no need to do this. You can fix it in the next PR.